src/docs/reference/modules/Tenants/README.md

```diff
@@ -21,4 +21,68 @@ E.g.
 `App_Data\Sites\Tenant1\wwwroot\robots.txt`
 
 `App_Data\Sites\Tenant2\wwwroot\robots.txt`
- 
\ No newline at end of file
+
+## Feature Profiles
```

Comment:
Shouldn't this be referenced here? https://github.com/OrchardCMS/OrchardCore/pull/9178/files#diff-0fbeec52dce3fd9818da84bd0a6ec5e1e86ca355d815edd5bd83d124724d2a19R80